### ISSUE TITLE
NTR: Horizontal position of testicle (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7963]
New term request Horizontal position of testicle (seidel_190206021847): High-riding testicle; testis; testes; lie

**Comment by pnrobinson** [comment]
@MickeySegal
there seem to be two relevant phenotypic features,
https://www.paedsurology.com/clinical-conditions/bell-clapper-testes/#:~:text=The%20term%20Bell%20Clapper%20Testes,abnormality%20normally%20affects%20both%20testes.
and
high riding testicle with torsion
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6863585/

Probably it is best to differentiate between them. Could you provide more context and or thoughts about how to do this?

**Comment by MickeySegal** [comment]
We used the term only for the disease "Testicular torsion", favoring the second example you used, but our language focused on the observable finding of the horizontal testicle, favoring the first example.

But it may be overkill to seek to separate the readily observable horizontal position from the underlying but not so readily observable anatomy:
https://kidshealth.org/en/teens/torsion.html
"Most cases of testicular torsion (also called testis torsion) affect guys who have a condition called a bell clapper deformity. In most males, the testicles are attached to the scrotum, making it hard for them to twist. In males who have the bell clapper deformity, the testicles are unsecured and can move and twist in the scrotum."

**Comment by pnrobinson** [comment]
So possibly it is better to label the term "bell clapper deformity" and to define it describing the above?

**Comment by MickeySegal** [comment]
That would be fine, but since few will recognize the term Bell Clapper we will use "Horizontal orientation" (instead of position) as the main term.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000839
- label: Horizontal position of testicle
- parents: [Abnormal testis morphology (HP:0000035)](https://hpo.jax.org/app/browse/term/HP:0000035)
- pmid: PMID:22606621
- def: The Bell Clapper deformity is one of the causes for the testicular torsion. In this condition, testis lacks a normal attachment to tunica vaginalis and hangs freely, causing it to sit horizontally rather than vertically in the scrotum.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7963
- synonyms: Bell-clapper deformity(NONE)
